lib/opam_cmd.ml

```diff
@@ -122,7 +122,10 @@ let classify_package ~package ~dev_repo ~archive ~pins () =
             | true ->
                 let base_repo = String.cuts ~empty:false ~sep:"#" archive |> List.hd in
                 (`Git base_repo, tag)
-            | false -> (`Unknown host, tag) )
```

Comment:
You removed the only use of `Unknown`, you can remove it in `package_is_valid` below and in `types.ml`

Reply:
Removed these entries !